.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,82 @@
+name: Bug report
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Fill in this template to report a bug
+
+title: "[BUG] "
+
+labels: ["bug"]
```

```suggestion
labels: ["type:bug"]
```

We're using this label so far to say "it's a bug" and then can specify further down with e.g. `"bug:functional"`.